platform/mellanox/mlnx-platform-api/sonic_platform/eeprom.py

```diff
@@ -46,17 +49,23 @@ class Eeprom(eeprom_tlvinfo.TlvInfoDecoder):
     EEPROM_DECODE_OFFSET = 0
     EEPROM_DECODE_CONTENT = 2
 
+    def _wait_for_eeprom_symlink(self):
```

Comment:
@tomer-israel please move this method behind the conductor's definition (before `def _load_eeprom(self)`): let's not break the code style